### ISSUE TITLE
NTR, pulmo: Nonspecific interstitial pneumonia (NSIP)

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6438]
**Preferred term label:**
temporal uniform pulmonary inflammation (maybe better to differentiate between histological term and disease term?)

**Synonyms**
Nonspecific interstitial pneumonia (NSIP)

**Definition (free text, please give PubMed ID)**
temporal uniform (all lesions are in the same stage of evolution) pattern of diffuse inflammatory process, mostly symmetric over the entire lung, involving mainly the alveolar septa. 

seperated in cellular and fibrotic subtypes
childterms? 
* fibrotic non-specific interstitial pneumonia: more common, worse prognosis, interstitial thickening is due to uniform dense or loose fibrosis and mild chronic inflammation; 

* cellular non-specific interstitial pneumonia: less common, better prognosis, interstitial thickening is mainly due to infiltration of inflammatory cells and type II pneumocyte hyperplasia. 

**Parent term (use hpo.jax.org/app)**
Interstitial pneumonitis HP:0006515

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:91364   (Disorder)   Non-specific interstitial pneumonia

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
@GrieseAG can we discuss the preferred term label?

**Comment by pnrobinson** [comment]
added
